src/component/LayoutCorporate.js

```diff
@@ -5,73 +5,100 @@ import { Link } from 'gatsby';
 import Search from './corporate/search';
 
 const WazoHeader = () => {
-  return <>
-    <div className="wazo-header">
-      <div className="container">
-        <div className="site-logo">
-          <a href="/" title="Wazo" rel="home">
-            <img className="header-image" alt="Wazo" src="https://i1.wp.com/wazo.io/wp-content/uploads/2020/03/WAZO-LogoWAZO-transparent-2.png?resize=300%2C79&ssl=1" title="Wazo" />
-          </a>
-        </div>
-
-        <nav id="site-navigation" className="has-sticky-branding main-navigation stuckElement" itemType="https://schema.org/SiteNavigationElement">
-          <div className="inside-navigation grid-container grid-parent">
+  return (
+    <>
+      <div className="wazo-header">
+        <div className="container">
+          <div className="site-logo">
+            <a href="/" title="Wazo" rel="home">
+              <img
+                className="header-image"
+                alt="Wazo"
+                src="https://wazo.io/wp-content/uploads/2023/04/logo-wazo.svg"
```

Comment:
why depend on other website for our assets instead of hosting/serving them locally ?